### DISCUSSION TITLE
Microsoft Xbox 360 Development Kit Hardware · RetroReversing

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#125] [Post Comments]
# Microsoft Xbox 360 Development Kit Hardware · RetroReversing

Learn about RetroReversing, the home of retro game reverse engineering enthusiasts. Get resources, tutorials, and tools for analyzing vintage technology and join the community today!

https://www.retroreversing.com/microsoft-xbox-360-development-kit/

**Comment by Ibram-RIMS** [comment]
Does the Xbox 360 XeDK Beta 1&2 have double of the ram from retail Xbox 360 or it doesn’t?

**Comment by Ibram-RIMS** [comment]
Does the Xbox 360 XeDK Beta 1&2 have double of the ram from the retail Xbox 360 or it’s doesn’t have double of the ram?

**Comment by Ibram-RIMS** [comment]
Also can you compete there specs to the retail Xbox 360 isn’t in enough information/specs for all of the Xbox 360 dev kit version like there port what like look inside and etc.
